Core/clim-core/frames.lisp

```diff
@@ -879,6 +873,24 @@ documentation produced by presentations.")
                     (push option (getf all-values :other-options)))))
       (alexandria:remove-from-plist all-values :pane))))
 
+(defmethod find-pane-for-frame ((fm frame-manager) (frame application-frame))
+  (make-pane-1 fm frame 'top-level-sheet-pane
+               :name (frame-name frame)
+               :pretty-name (frame-pretty-name frame)
+               ;; sheet is enabled from enable-frame
+               :enabled-p nil))
+
+(defmethod generate-panes (fm (frame application-frame))
+  (with-look-and-feel-realization (fm frame)
+    (unless (frame-panes-for-layout frame)
+      (setf (frame-panes-for-layout frame)
+            `((single-pane . ,(make-clim-interactor-pane :name 'single-pane)))))
+    (let ((single-pane
+            (alexandria:assoc-value (frame-panes-for-layout frame)
+                                    'single-pane :test #'eq)))
+      (setf (frame-panes frame) single-pane)))
```

Comment:
Maybe add a comment explaining that `make-clim-interactor-pane` may wrap the `single-pane` instance in other panes and therefore the lookup in `frame-panes-for-layout` is needed.

Reply:
the lookup in frame-panes-for-layout is because there because we create the interactor only once (in the `unless` form), `make-clim-interactor-pane` returns the wrapper pane as the first value (stream is returned as the second value).